ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -364,6 +385,13 @@ - (void)swizzled_notifyWillDisappear
   [_sharedTransitionManager screenRemovedFromStack:(UIView *)self];
 }
 
+- (void)swizzled_notifyDisappear
+{
+  // call original method from react-native-screens, self == RNSScreen
+  [self swizzled_notifyDisappear];
+  [_sharedTransitionManager screenDidDisappear];
```

Comment:
Is this code still needed? I didn't see a corresponding code on Android? Maybe it'd be worth adding event to screens instead of swizzling here similarly to how you added progress cancel event?

Reply:
I removed it